examples/io/plot_read_events.py

```diff
@@ -0,0 +1,77 @@
+"""
+.. _ex_read_events:
```

Comment:
sphinx gallery automatically adds a link to the file based on it's name. Better to do this than manually setting a link name.